packages/edit-site/src/components/sidebar-dataviews/index.js

```diff
@@ -45,13 +45,13 @@ export default function DataViewsSidebarContent() {
 	const {
 		params: { path, activeView = 'all' },
 	} = useLocation();
-	if( ! path || ! DEFAULT_VIEWS[ path ]) {
+	if( ! path || path != '/pages') {
```

Comment:
I think this check is useless no?

Reply:
No for some reason without the check the app crashed as soon as we clicked on a page.

Reply:
I think that's probably a subtle issue elsewhere.